.kitchen.cloud.yml

```diff
@@ -74,6 +77,8 @@ suites:
     run_list:
       - role[openstack_ops_database]
       - role[openstack_controller]
+      - recipe[openstack-integration-test::setup]
+      - recipe[scl]
```

Comment:
Oh, it is in the metadata. If you're only calling it from .kitchen.cloud.yml, and not anywhere in `recipes/`, it shouldn't be in metadata.